### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by solbage** [community/community#34459] [Other Feature Feedback, Questions, & Ideas]
hello. let me introduce  myself.
my nickname is solz. I'm a elementary school teacher.

**Comment by radar-blip** [comment]
Hello solz, welcome!

**Reply by InacioCarvalhoOliveira** [reply]
you're welcome @Solbage, w

**Comment by jfullstackdev** [comment]
you can also post it here for more visibility

https://github.com/orgs/community/discussions/23204
